config/jobs/kubernetes/sig-release/release-branch-jobs/1.28.yaml

```diff
@@ -0,0 +1,2200 @@
+        - name: KUBE_VERIFY_GIT_BRANCH
+          value: release-1.28
+        - name: REPO_DIR
+          value: /workspace/k8s.io/kubernetes
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.28
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: "7"
+            memory: 12Gi
+          requests:
+            cpu: "7"
+            memory: 12Gi
+        securityContext:
+          privileged: true
+  - always_run: true
+    branches:
+    - release-1.28
+    cluster: k8s-infra-prow-build
+    context: pull-kubernetes-verify
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+    name: pull-kubernetes-verify
+    path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - args:
+        - ./hack/jenkins/verify-dockerized.sh
+        command:
+        - runner.sh
+        env:
+        - name: EXCLUDE_TYPECHECK
+          value: "y"
+        - name: EXCLUDE_GODEP
+          value: "y"
+        - name: KUBE_VERIFY_GIT_BRANCH
+          value: release-1.28
+        - name: REPO_DIR
+          value: /workspace/k8s.io/kubernetes
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.28
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: "7"
+            memory: 12Gi
+          requests:
+            cpu: "7"
+            memory: 12Gi
+        securityContext:
+          privileged: true
+  - always_run: false
+    branches:
+    - release-1.28
+    cluster: k8s-infra-prow-build
+    context: pull-kubernetes-verify-strict-lint
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+    name: pull-kubernetes-verify-strict-lint
+    optional: true
+    path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - args:
+        - /bin/sh
+        - -c
+        - if [ -x hack/verify-golangci-lint-pr.sh ]; then hack/verify-golangci-lint-pr.sh; fi
+        command:
+        - runner.sh
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.28
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: "7"
+            memory: 12Gi
+          requests:
+            cpu: "7"
+            memory: 12Gi
+  - always_run: false
+    branches:
+    - release-1.28
+    context: pull-kubernetes-e2e-capz-windows
+    decorate: true
+    extra_refs:
+    - base_ref: release-1.10
+      org: kubernetes-sigs
+      path_alias: sigs.k8s.io/cluster-api-provider-azure
+      repo: cluster-api-provider-azure
+      workdir: true
+    - base_ref: master
+      org: kubernetes-sigs
+      path_alias: sigs.k8s.io/cloud-provider-azure
+      repo: cloud-provider-azure
+    labels:
+      preset-azure-anonymous-pull: "true"
+      preset-azure-capz-sa-cred: "true"
+      preset-azure-cred-only: "true"
```

Comment:
This line is causing the unit tests to fail:
```
jobs_test.go:556: "pull-kubernetes-e2e-capz-windows" defined in "../../jobs/kubernetes/sig-release/release-branch-jobs/1.28.yaml" may not use secret "azure-cred" in "default" cluster
```

Reply:
@saschagrunert This is was added by running`go run main.go --job-config $GOPATH/src/k8s.io/test-infra/config/jobs/kubernetes/sig-release --version 1.28 --go-version 1.20.6 `. Not sure what changed. Also `pull-kubernetes-e2e-capz-windows` in 1.27 has to those presets.

Reply:
Updated the test in https://github.com/kubernetes/test-infra/pull/30208/commits/4a0aa14f0dd9c6359ef0917a266f3a04f869d996 to allow presubmits with azure creds.